scss/_patterns_side-navigation.scss

```diff
@@ -52,6 +52,11 @@
     }
   }
 
+  .p-side-navigation__status {
+    margin-left: auto;
+    padding-left: $spv-outer--small;
```

Comment:
This should be $spv-inner--small (because it is inside, not outside of the element)